polkadot-parachains/statemine/src/weights/pallet_collator_selection.rs

```diff
@@ -26,46 +45,66 @@ use sp_std::marker::PhantomData;
-		(18_481_000 as Weight)
+	// Storage: CollatorSelection Invulnerables (r:0 w:1)
+	fn set_invulnerables(b: u32, ) -> Weight {
+		(17_818_000 as Weight)
 			// Standard Error: 0
-			.saturating_add((67_000 as Weight).saturating_mul(b as Weight))
+			.saturating_add((48_000 as Weight).saturating_mul(b as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+	// Storage: CollatorSelection DesiredCandidates (r:0 w:1)
 	fn set_desired_candidates() -> Weight {
-		(16_376_000 as Weight).saturating_add(T::DbWeight::get().writes(1 as Weight))
+		(15_850_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+	// Storage: CollatorSelection CandidacyBond (r:0 w:1)
 	fn set_candidacy_bond() -> Weight {
-		(17_031_000 as Weight).saturating_add(T::DbWeight::get().writes(1 as Weight))
+		(16_377_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
-	fn register_as_candidate(c: u32) -> Weight {
-		(72_345_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((197_000 as Weight).saturating_mul(c as Weight))
-			.saturating_add(T::DbWeight::get().reads(4 as Weight))
+	// Storage: CollatorSelection Candidates (r:1 w:1)
+	// Storage: CollatorSelection DesiredCandidates (r:1 w:0)
+	// Storage: CollatorSelection Invulnerables (r:1 w:0)
+	// Storage: Session NextKeys (r:1 w:0)
+	// Storage: CollatorSelection CandidacyBond (r:1 w:0)
+	// Storage: CollatorSelection LastAuthoredBlock (r:0 w:1)
+	fn register_as_candidate(c: u32, ) -> Weight {
+		(76_016_000 as Weight)
+			// Standard Error: 1_000
+			.saturating_add((151_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
-	fn leave_intent(c: u32) -> Weight {
-		(55_446_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((153_000 as Weight).saturating_mul(c as Weight))
+	// Storage: CollatorSelection Candidates (r:1 w:1)
+	// Storage: CollatorSelection LastAuthoredBlock (r:0 w:1)
+	fn leave_intent(c: u32, ) -> Weight {
+		(76_011_000 as Weight)
+			// Standard Error: 1_000
+			.saturating_add((202_000 as Weight).saturating_mul(c as Weight))
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
+	// Storage: System Account (r:2 w:2)
+	// Storage: System BlockWeight (r:1 w:1)
+	// Storage: CollatorSelection LastAuthoredBlock (r:0 w:1)
 	fn note_author() -> Weight {
-		(71_828_000 as Weight)
+		(67_925_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
-	fn new_session(r: u32, c: u32) -> Weight {
+	// Storage: CollatorSelection Candidates (r:1 w:1)
+	// Storage: CollatorSelection LastAuthoredBlock (r:1000 w:1)
+	// Storage: System Account (r:1 w:1)
+	// Storage: CollatorSelection Invulnerables (r:1 w:0)
+	// Storage: System BlockWeight (r:1 w:1)
```

Comment:
It's a bit odd that System BlockWeight is not whitelisted for the benchmarks, but not sure it's worth it to dive deep into this :shrug: